src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitService.kt

```diff
@@ -147,6 +154,15 @@ class VisitService(
     )
   }
 
+  private fun saveLegacyData(visit: Visit, leadPersonId: Long) {
```

Comment:
Should we rename all of these creates to save?

Reply:
Yep I tried to make a distinction between the other methods that created objects and did not save!